docs/io.cozy.account_types.md

```diff
@@ -0,0 +1,30 @@
+- [Cozy account types doctype](#cozy-account-types-doctype)
+  - [`io.cozy.account_types`](#iocozyaccounttypes)
+
+# Cozy account types doctype
+
+## `io.cozy.account_types`
+
+The `io.cozy.account_types` can be used for two main purposes:
+
+- It can contain secret configuration parameters needed by the webapp/konnectors
+  (API Tokens, API URLS, ...). Every time a konnector job is executed, the stack
+  can give some of these secret parameters to the konnector
+
+- It can contain an OAuth configuration for a service. It makes possible for the
+  stack to talk to external services without allowing the konnector to directly
+  access the outside world
```

Comment:
No, the konnector can still access the outside world, and the OAuth provider in particular. It is just a way to simplify the development of konnectors for OAuth providers, and it avoid race conditions when the same account can be used by several konnectors (I think of the example of google).